patternlab/styleguide/source/_patterns/01-atoms/09-media/figure--dataviz.twig

```diff
@@ -0,0 +1,31 @@
+{% if figure.dataviz %}
+  {# Render figure.content, which will be the main content of the figure #}
+  {# Important: we wrap this in an `if` statement to allow consuming systems
+     to either use figure.image or figure.content (if they prefer to externally
+     render figure.content
+   #}
+  {% set align = figure.align|default('left') %}
+  {% set id = figure.dataviz.id %}
+  {# TODO:  SET SIZE VALUES #}
+  {# Set default size to large. #}
+  {% set size = "ma__figure--" ~ figure.dataviz.size|default('large') %}
+  {% set class = size ~ ' ' ~ 'ma__dataviz' %}
+  {% set content %}
+    <div id="{{ figure.dataviz.id }}"></div>
+    <script type="text/javascript">
+      window.addEventListener("DOMContentLoaded", function() {
+        new tableau.Viz(document.getElementById("{{ figure.dataviz.id }}"), "{{ figure.dataviz.url }}");
+      });
+    </script>
+  {% endset %}
+  {% set figure = figure|merge({
+    'content': content,
+  }) %}
+{% endif %}
+{% include "@atoms/09-media/figure.twig" with {
+  figure: figure|merge({
+    'align': align,
+    'id': id,
```

Comment:
`id` doesn't seem to be used in `figure.twig`, so I don't think we want/need to pass it through.

Reply:
Implemented in #971.